axelor-account/src/main/resources/reports/AccountingReportType6.rptdesign

```diff
@@ -2135,11 +2141,10 @@ else  {  0.00 }</expression>
                                         <property name="borderRightWidth">thin</property>
                                         <property name="paddingTop">0pt</property>
                                         <property name="paddingBottom">0pt</property>
-                                        <text-data id="4264">
+                                        <data id="4264">
                                             <property name="fontWeight">bold</property>
-                                            <expression name="valueExpr">row["partner_seq"] + " - " + row["partner_name"]</expression>
-                                            <property name="contentType">html</property>
-                                        </text-data>
```

Comment:
Remove 'partner_seq' from selection query if its not used anywhere else in this report